recipes-graphics/clutter/clutter-1.0_%.bbappend

```diff
@@ -1,5 +1,5 @@
-PACKAGECONFIG_imxgpu2d ??= " \
+PACKAGECONFIG_imxgpu3d ??= " \
```

Comment:
Can you explain why we should use it with gpu3d instead of gpu2d before?
Are both working? If so, can't we have both options available with 1 preferred option?

Reply:
I think it ought to use imxgpu which implies it has any GPU. Will rework.

Reply:
Clutter now depends on imxgpu3d since imxgpu2d is no longer used as "any GPU" selector.
See following comment for details:
imx-base.inc: Add imxgpu to MACHINEOVERRIDES_EXTENDER
https://github.com/OSSystems/meta-freescale/commit/98c1cde6beb7183f84ef2e247e1e2b5ba14a1ede